applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/app.overlay

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* Redefine the partition map. */
+&rram0 {
+	/delete-node/ partitions;
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+		app_partition: partition@0 {
+			label = "application";
+			reg = <0x0 0x177000>;
+		};
+
+		storage_partition: partition@177000 {
+			label = "storage";
+			reg = <0x177000 0x6000>;
+		};
+	};
+};
+
+/ {
+	chosen {
+		zephyr,flash = &rram0;
+		zephyr,code-partition = &app_partition;
+	};
+
+	/* Disable leds and redefine them to align configuration with CAF LEDs requirements. */
+	leds {
+		status = "disabled";
+	};
+
+	gpioled0 {
```

Comment:
Do I understand correctly that we cannot use default leds because we need to have one led per node?
Maybe do not disable but remove original leds node?

 Also consider using default leds names: led0_g: led_0 -> led0: led_0

Reply:
Yes, that's the reason behind disabling the node. The node is currently disabled (not removed) for other configuration so I would disable it also here for consistency.
I would prefer to stick to the current naming scheme too (also for consistency with existing configurations - e.g. `nrf52833dk_nrf52820`).